src/pages/index.css

```diff
@@ -0,0 +1,22 @@
+.overlayWithCircle {
+    position: absolute;
+    overflow-x: hidden;
+    overflow-y: hidden;
+    top: 0;
+    left: 0;
+    height: 100vh;
+    width: 100%;
+    z-index: -1;
+}
+
+.overlayWithCircle:after {
+    content: '';
+    position: absolute;
+    right: -15%;
+    top: 13%;
+    border-radius: 100%;
+    width: 60%;
+    height: 0;
+    padding-bottom: 60%;
+    box-shadow: 0 0 0 3000px #005550;
+}
```

Comment:
Dette er trikset for å få til sirkelen. En sirkel med en kjempebred, sentrert box-shadow. Animeringen går ut på å øke `width`, men for å holde sirkelen i sentrum, må også `right` og `top` flyttes på (ellers ville sirkelen flyttet seg ned til venstre samtidig som den ble større). 
`Padding-bottom` settes til det samme som `width` for å holde ratio på sirkelen uten å måtte kalkulere høyden.